app/controllers/ToDo/ShowController.js

```diff
@@ -41,7 +41,9 @@ class ShowController {
             params: { id: todoId }
         } = request;
 
-        const todo = await this.todoRepository.findById(todoId);
+        const todo = await this.todoRepository.findOne({
```

Comment:
use findById but add anohter `where` condition, probably you have to change `findById` in AbstractRepo and use `deepMerge`